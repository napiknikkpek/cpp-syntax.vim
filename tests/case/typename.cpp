template <typename T>
struct F;
template <typename B, typename = std::enable_if_t<std::is_base_of_v<B, D>>>
struct F;
typename X::A::asdf;
template <typename... Args>
void foo(Args&&... args) {
  fn(args...);
}
