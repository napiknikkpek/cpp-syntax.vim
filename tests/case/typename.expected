aaaaaaaa..aaaaaaaa.t.
aaaaaa.t.
aaaaaaaa..aaaaaaaa.t..aaaaaaaa...ttt..ttttttttttt.ttt..tttttttttttt.x..x...
aaaaaa.t.
aaaaaaaa.t..t..tttt.
aaaaaaaa..aaaaaaaa....tttt.
zzzz.fff.tttt......xxxx...
..ff.xxxx.....
.
